packages/tools/networking/openssh/plan.sh

```diff
@@ -4,23 +4,28 @@ pkg_version="9.1p1"
 pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
 pkg_description="Provides OpenSSH client and server."
 pkg_upstream_url="https://www.openssh.com/"
-pkg_license=('bsd')
+pkg_license=('SSH-OpenSSH')
 pkg_source="http://mirror.wdc1.us.leaseweb.net/openbsd/OpenSSH/portable/${pkg_name}-${pkg_version}.tar.gz"
 pkg_shasum="19f85009c7e3e23787f0236fbb1578392ab4d4bf9f8ec5fe6bc1cd7e8bfdd288"
 
+# Select the correct openssl dep
+case $pkg_target in
+aarch64-*)
+	openssl_dep="core/openssl11"
+	;;
+x86_64-*)
+	openssl_dep="core/openssl"
+	;;
+esac
```

Comment:
This complexity is not needed anymore, we only want openssl 3. We need to upgrade the openssh package as well, as the older version does not build on arm with openssl 3

Reply:
removed this block and updated openssh